src/lib/notificationBroadcasterWebsocket.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const _ = require('lodash')
+const EventEmitter = require('events').EventEmitter
+const transferDictionary = require('five-bells-shared').TransferStateDictionary
+const transferStates = transferDictionary.transferStates
+const isTransferFinalized = require('./transferUtils').isTransferFinalized
+const convertToExternalTransfer = require('../models/converters/transfers')
+  .convertToExternalTransfer
+const getFulfillment = require('../models/db/fulfillments').getFulfillment
+const convertToExternalFulfillment = require('../models/converters/fulfillments')
+  .convertToExternalFulfillment
+
+class NotificationBroadcaster extends EventEmitter {
+  constructor (log, config) {
+    super()
+    this.log = log
+    this.config = config
+  }
+
+  * sendNotifications (transfer, transaction) {
+    const affectedAccounts = _([transfer.debits, transfer.credits])
+      .flatten().pluck('account').value()
+    affectedAccounts.push('*')
+
+    // Prepare notification for websocket subscribers
+    const notificationBody = {
+      resource: convertToExternalTransfer(transfer)
+    }
+
+    // If the transfer is finalized, see if it was finalized by a fulfillment
+    let fulfillment
```

Comment:
As I mentioned on #277, unless we have a good reason for including the condition fulfillment in the notification, I'd advocate (in the interests of performance) removing this extra DB read. The client may fetch the fulfillment on an as-needed basis via the `GET /transfers/:id/fulfillment` endpoint.